.travis.yml

```diff
@@ -21,8 +21,9 @@ install:
 script:
   - npm t -- --reporter=dots
   - npm run docs:api -- --validate-only
-after_success:
```

Comment:
Fail in section `after_sucess` does not change the status of the build (https://github.com/travis-ci/travis-ci/issues/758) so we need to move the docs command to `script` section (that is able to mark the build as failed).